Apps/FlightSoftware/Watchdog/src/main.c

```diff
@@ -150,6 +153,8 @@ int main(void) {
     // TODO: debug
     ipudp_send_packet("hello, world!\r\n", 15); // @suppress("Invalid arguments")
 
+    I2C_Sensors__Readings i2cReadings = { 0 }; // will hold i2c data

```

Comment:
There is also a global named this.

Reply:
Yup - my bad on not siding with one of global or passing as argument for i2cReadings. I can make it one or the other just let me know which is preferable

Reply:
Keeping this in